RFS/src/main/java/com/rfs/common/DocumentReindexer.java

```diff
@@ -31,13 +32,15 @@ public Mono<Void> reindex(
             .buffer(numDocsPerBulkRequest) // Collect until you hit the batch size
             .doOnNext(bulk -> logger.info("{} documents in current bulk request", bulk.size()))
             .map(this::convertToBulkRequestBody)  // Assemble the bulk request body from the parts
+            // Retry on failures accumulating requests
+            .retryWhen(Retry.backoff(3, Duration.ofSeconds(1)).maxBackoff(Duration.ofSeconds(5)))
```

Comment:
This seems like it needs to be AFTER the sendBulkRequest call, where it was.  Why did you rotate it upward?

Reply:
Discussed offline, the actual retry mechanism is inside sendBulkRequest. I've since removed this